package.json

```diff
@@ -90,5 +90,8 @@
     "q": "1.4.1",
     "u2f-api-polyfill": "0.4.1",
     "underscore": "1.8.3"
+  },
+  "optionalDependencies": {
+    "fsevents": "*"
```

Comment:
Gotcha, yeah the shrinkwrap thing is another problem that we'll need to fix now that npm5 is out, for now let's just go with the optional route.